File: benchmarks/cross_runtime/loop_arith.py
# Loop apertado: despacho de bytecode e aritmetica inteira, sem alocacao.
def main():
    i = 0
    s = 0
    while i < 3000000:
        s = (s + i * 3) % 1000003
        i = i + 1
    print(f"CHECKSUM:{s}")


main()

File: benchmarks/cross_runtime/loop_arith.lua
-- Loop apertado: despacho de bytecode e aritmetica inteira, sem alocacao.
local function main()
  local i, s = 0, 0
  while i < 3000000 do
    s = (s + i * 3) % 1000003
    i = i + 1
  end
  print("CHECKSUM:" .. s)
end

main()
